lib/Core/Memory.cpp

```diff
@@ -9,13 +9,12 @@
 
 #include "Memory.h"
 
+#include "klee/Core/BitArray.h"
+#include "klee/Expr/ArrayCache.h"
+#include "klee/Expr/Expr.h"
+#include "klee/Solver/Solver.h"
+#include "klee/Support/ErrorHandling.h"
 #include "Context.h"
```

Comment:
Local header `Context.h` should come before these if are allowing library local includes (which this file already has).